tests/model_fields/models.py

```diff
@@ -319,6 +321,27 @@ class PersonTwoImages(models.Model):
                                   width_field='headshot_width')
 
 
+class JSONModel(models.Model):
+    value = models.JSONField()
+
+
+class CustomDecoder(json.JSONDecoder):
+    def __init__(self, object_hook=None, *args, **kwargs):
+        return super().__init__(object_hook=self.object_hook, *args, **kwargs)
+
+    def object_hook(self, dct):
+        try:
+            dct['uuid'] = uuid.UUID(dct['uuid'])
+        except KeyError:
+            pass
+        finally:
```

Comment:
This doesn't need to be under `finally`

Reply:
Yeah, I originally didn't do so, but I figured it looked nice to have the statements indented at the same level.

Reply:
It's better to have a clearer single `return` from a function.